test/cli-e2e.spec.js

```diff
@@ -256,7 +256,7 @@ describe('cli e2e', () => {
     });
   });
 
-  it('should pass chunks from the build stats to selenium', () => {
+  it('should pass chunks from the build stats to selenium', (done) => {
```

Comment:
This spec was missing the `done` function (unrelated to the features of this pull request).